birdhouse/config/jupyterhub/default.env

```diff
@@ -5,7 +5,7 @@
 # are applied and must be added to the list of DELAYED_EVAL.
 
 export JUPYTERHUB_DOCKER=pavics/jupyterhub
-export JUPYTERHUB_VERSION=1.4.0-20210506
+export JUPYTERHUB_VERSION=4.0.2-20230816
```

Comment:
Just curious, this upgrade is needed or just nice to have?  Maybe explain the reason for the upgrade of the JupyterHub in the changelog as well.